include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -310,10 +310,19 @@ typedef struct OrtCUDAProviderOptions {
 /// Options for the ROCM provider that are passed to SessionOptionsAppendExecutionProvider_ROCM
 /// </summary>
 typedef struct OrtROCMProviderOptions {
-  int device_id;                      // hip device with id=0 as default device.
-  int miopen_conv_exhaustive_search;  // miopen conv algo exhaustive search option
-  size_t gpu_mem_limit;               // default hip memory limitation to maximum finite value of size_t.
-  int arena_extend_strategy;          // default area extend strategy to KNextPowerOfTwo.
+  int device_id;                                  // hip device with id=0 as default device.
```

Comment:
Having these structs defined in the C header was a bad idea to begin with if you intend to package this provider and expose it as a C API. Is there any intention to package this?